trinity/server.py

```diff
@@ -181,25 +181,42 @@ def _make_request_server(self) -> BaseRequestServer:
             HandshakeFailure,
             asyncio.IncompleteReadError,
         )
+
+        def cleanup_reader_and_writer():
```

Comment:
[CI linting error](https://circleci.com/gh/ethereum/py-evm/119237?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link): function is missing a type annotation.